CrosswordMe.py

```diff
@@ -76,15 +77,15 @@ def create_puzzle(words):
     # creating the puzzle
     puzzle = [["$" for x in range(column_count)] for y in range(line_count)] #Creates array filled with "$"
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `python`.*

The issue can be fixed by applying the following patch:

```diff
--- a/CrosswordMe.py
+++ b/CrosswordMe.py
@@ -75,11 +75,11 @@
 
 def create_puzzle(words):
     # creating the puzzle
-    puzzle = [["$" for x in range(column_count)] for y in range(line_count)] #Creates array filled with "$"
+    puzzle = [["$" for x in range(column_count)] for y in range(line_count)]  # Creates array filled with "$"
 
-    for word_index in range(len(words)): #loops through every word in the words list
+    for word_index in range(len(words)):  # loops through every word in the words list
 
-        word_direction = randint(0,2)
+        word_direction = randint(0, 2)
         if word_direction == 0:
             puzzle = write_horizontal(words, word_index, puzzle)
         elif word_direction == 1:
```